testing/jormungandr-integration-tests/src/jormungandr/rest/v1/utils/settings.rs

```diff
@@ -0,0 +1,137 @@
+use std::num::{NonZeroU64, NonZeroU32};
+use crate::startup;
+use chain_impl_mockchain::fee::{LinearFee, PerCertificateFee, PerVoteCertificateFee};
+use chain_impl_mockchain::rewards::{CompoundingType, Parameters, Limit};
+use jormungandr_automation::jormungandr::ConfigurationBuilder;
+use jormungandr_lib::interfaces::{Block0Configuration, TaxType, Ratio};
+use jormungandr_lib::interfaces::{SettingsDto, RewardParams, BlockchainConfiguration};
+use jormungandr_lib::time::SystemTime;
+use rstest::*;
+
+fn get_parameters_from_blockchain_config(blockchain_configuration: &BlockchainConfiguration) -> Option<Parameters> {
+    let reward_param =
+        match blockchain_configuration.reward_parameters{
+            None => return None,
+            Some(r) => r
+        };
+
+    let reward_drawing =
+         match blockchain_configuration.reward_constraints.reward_drawing_limit_max{
+            None => Limit::None,
+            Some(r) => Limit::ByStakeAbsolute(r.into())
+        };
+    //TODO use map?
+    let pool_participation =
+        match blockchain_configuration.reward_constraints.pool_participation_capping
+         {
+            None => None,
+            Some(p) => Some((p.min, p.max))
+        };
+
+    match reward_param {
+        RewardParams::Linear {
+            constant,
+            ratio,
+            epoch_start,
+            epoch_rate,
+        } => Some(Parameters {
+            initial_value: constant,
+            compounding_ratio: ratio.into(),
+            compounding_type: CompoundingType::Linear,
+            epoch_rate,
+            epoch_start,
+            reward_drawing_limit_max: reward_drawing,
+            pool_participation_capping: pool_participation,
+        }),
+        RewardParams::Halving {
+            constant,
+            ratio,
+            epoch_start,
+            epoch_rate,
+        } => Some(Parameters {
+            initial_value: constant,
+            compounding_ratio: ratio.into(),
+            compounding_type: CompoundingType::Halvening,
+            epoch_rate,
+            epoch_start,
+            reward_drawing_limit_max: reward_drawing,
+            pool_participation_capping: pool_participation,
+        }),
+    }
+}
+
+fn get_settings_from_block0_configuration(block0_configuration: &Block0Configuration) -> SettingsDto{
```

Comment:
i think it would be bettter to extent Block0Configuration trait and add this method there.
https://github.com/input-output-hk/jormungandr/blob/master/testing/jormungandr-automation/src/testing/block0.rs#L39

then you can call:
```
blockchain_configuration.rewards_parameters()
```